docs/components/modal.md

```diff
@@ -130,9 +130,13 @@ Change the [appearance](/components/button.html#props) of the cancel button.
 
 Use this to hide the built-in cancel button (`false` by default).
 
-### showDismissIcon
+### hideDismissIcon
 
-Boolean for whether or not to display the 'X' in the upper right corner to dismiss the dialog (`false` by default).
+When using the `header-image` slot we display a dismiss 'X' button in the upper right corner of the dialog. Set this prop to `true` to hide the.
```

Comment:
this section is not complete. 

When using the `header-image` slot we display a dismiss 'X' button in the upper right corner of the dialog. Set this prop to `true` to hide the button (`false` by default).

Reply:
@kaiarrowood these will need to go to `main` as well 😬